IdentityCore/src/validation/MSIDAADAuthority.m

```diff
@@ -223,7 +223,11 @@ - (NSString *)enrollmentIdForHomeAccountId:(NSString *)homeAccountId
 
 - (nonnull NSString *)telemetryAuthorityType
 {
+#if !EXCLUDE_FROM_MSALCPP
     return MSID_TELEMETRY_VALUE_AUTHORITY_AAD;
+#else // MSAL CPP
+    return @"";
```

Comment:
Can this be avoided?

Reply:
I don't think so... This is not needed in MSAL CPP, and I will need to re-define an empty string in MSIDRequestTelemetryConstants...